### ISSUE TITLE
Firmware update potential issue

### ISSUE THREAD

**Issue by BenHarris** [lurume84/bling-wrt#2]
I was just having a look over the filesystem and noticed the `/root/utils/sm_update` file. It is worth noting that it appears a firmware update will wipe and apps/utils/web
```
rm -rf /root/apps
rm -rf /root/utils
rm -rf /www
```

Might be a nice idea to write an init script to automatically restore access in the event of an update. I'm not sure if they are automated?

**Comment by lurume84** [comment]
Yes, it was my next step to produce some init scripts to automate everything. I haven't invested more time in this repo but in case you are able to develop some of them I will happily accept pull requests. Thanks!
